types/src/front/lib/assistant.ts

```diff
@@ -286,6 +287,14 @@ const ANTHROPIC_DELIMITERS_CONFIGURATION = {
   ],
 };
 
+const ANTHROPIC_TOOL_USE_META_PROMPT =
+  `Immediately before using a tool, think for one short bullet point in \`<thinking>\` tags about ` +
+  `how it evaluates against the criteria for a good and bad tool use. ` +
+  `After using a tool, think for one short bullet point in \`<thinking>\` tags to evaluate ` +
```

Comment:
this one is important as well, otherwise the model will generate non chain of thought content before using the next tool, which looks weird in the response (this is my experience with sonnet).